trailblazer/store/crud/create.py

```diff
@@ -29,6 +29,7 @@ def add_pending_analysis(self, analysis_data: CreateAnalysisRequest) -> Analysis
             type=analysis_data.type,
             workflow_manager=analysis_data.workflow_manager,
             tower_workflow_id=analysis_data.tower_workflow_id,
+            is_visible=not analysis_data.is_hidden,
```

Comment:
why not using variables meaning the same thing both sites? It would be clearer if you named this parameter `is_hidden` instead of `is_visible`, as the one in the `CreateAnalysisRequest` is called `is_hidden`.

Reply:
In Trailblazer the column is called `is_visible` so the request's parameter should be called `is_visible`. But the only time we actually want to specify this parameter is when we want the analysis hidden. Maybe it would be clearer to name it `is_visible` but it would also complicate the default behaviour a bit, as `is_visible == True` should be handled the same way as `is_visible == None`.

Reply:
I mean, the database model has the columned named `is_visible` so the only thing that could be changed is the `CreateAnalysisRequest` object. But the value will pretty much only be passed when you want to hide analyses. But maybe it would be clearer if it was named `is_visible` there as well?

Reply:
ok it seems to extensive and out of scope, resolving